webapp/src/Form/Type/ContestType.php

```diff
@@ -39,28 +41,34 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'Yes' => true,
                 'No' => false,
             ],
+            'help' => 'Only John knows',
```

Comment:
Disable to delay the contest start. Enable again after setting a new start time.

Reply:
What effect has `delay the contest start` on the GUI? Will the coundown in the top right just disappear?

Reply:
Yes, it will display "countdown paused" or a similar message.